includes/wc-user-functions.php

```diff
@@ -468,7 +468,7 @@ function wc_get_customer_available_downloads( $customer_id ) {
 					home_url( '/' )
 				),
 				'download_id'         => $result->download_id,
-				'product_id'          => $product_id,
+				'product_id'          => $_product->id,
```

Comment:
@claudiosmweb,

Hi claudiosmweb I have clone of woocommerce which has version 2.5.0-dev. Now I want to take latest updated clone which has version 2.7.0-dev how can I do that would you please guide me so that I can contribute to latest development branch.

Thanks,